Registry/Registry.EM_COMMON

```diff
@@ -2447,6 +2451,7 @@ rconfig   integer     irr_ph              namelist,physics      max_domains    0
 rconfig   integer     sf_surface_physics  namelist,physics	max_domains    -1      rh       "sf_surface_physics"            ""      ""
 rconfig   integer     bl_pbl_physics      namelist,physics	max_domains    -1      rh       "bl_pbl_physics"                ""      ""
 rconfig   integer     tke_budget          namelist,physics      max_domains    0       rh       "tke_budget"                    ""      ""
+rconfig   integer     bl_mynn_tkebudget   namelist,physics      max_domains    0       irh       "bl_mynn_tkebudget"             ""      ""
```

Comment:
For simplicity, I'd prefer to remove bl_mynn_tkebudget, and then just use tke_budget to regulate both versions. In the call to the old scheme (in the pbl driver), we can use use "bl_mynn_tkebudget=tke_budget". Then remove the new mynn_tkebudget package below.